Documentation/monitoring.md

```diff
@@ -49,9 +49,27 @@ Once the Prometheus server is running, you can open a web browser and go to the
 echo "http://$(kubectl -n rook -o jsonpath={.status.hostIP} get pod prometheus-rook-prometheus-0):30900"
 ```
 
-You should now see the Prometheus monitoring website.  Click on `Graph` in the top navigation bar.  In the dropdown that says `insert metric at cursor`,
-select any metric you would like to see, for example `ceph_cluster_used_bytes`, followed by clicking on the `Execute` button.  Below the `Execute` button, ensure
-the `Graph` tab is selected and you should now see a graph of your chosen metric over time.
+##### You should now see the Prometheus monitoring website.  
```

Comment:
i think it would look more natural to use normal text instead of the 5th level title